### ISSUE TITLE
What if we train GNN to Vertex data?

### ISSUE THREAD

**Issue by FilipsFedotovs** [FilipsFedotovs/EDER-VIANN#64]
Create data with CNN training performance. (Validation Loss, Test loss, Accuracy etc. vs Epochs). How does it compare to CNN?
